queries/cmake/highlights.scm

```diff
@@ -0,0 +1,40 @@
+[
+ (quoted_argument)
+ (bracket_argument)
+] @string
+
+(variable_ref) @variable
+
+[
+ (bracket_comment)
+ (line_comment)
+] @comment
+
+[
+ "$ENV"
+ "$CACHE"
+] @variable.builtin
+
+(normal_command (identifier) @function)
+
+[
+ (function)
+ (endfunction)
+ (macro)
+ (endmacro)
+] @function
+
+[
+ (if)
+ (elseif)
+ (else)
+ (endif)
+] @conditional
+
+[
+ (foreach)
+ (endforeach)
+ (while)
+ (endwhile)
+] @repeat
+
```

Comment:
Do you plan to add highlights for things like `ON`, `OFF`, `TRUE`, `FALSE` and operators like `AND`, `OR`. This can also be done in a subsequent PR

Reply:
yes, I want to have a minimum working version first then I will implement these improvements late.

Reply:
But at least all character tokens like `(` `${` `}` should be included.

Reply:
Now, `ENV`, `CACHE`, `$`, `{`, `}`, `(`, and `)` are included.